R/norges-bank-utils.R

```diff
@@ -0,0 +1,67 @@
+#' request_sdmx_latest
+#' @noRd
+request_sdmx_latest <- function(currency, frequency, n_obs) {
+  api_path <- sprintf(
+    'data/EXR/%s.%s.NOK.SP?format=sdmx-json&lastNObservations=%s&locale=en',
+    frequency, currency, n_obs
+  )
+  req <- request(nbank_url) |> req_url_path_append(api_path)
+  send_query(req)
+}
+
+#' request_sdmx
+#' @noRd
+request_sdmx <- function(currency, frequency, start, end) {
+  api_path <- sprintf(
+    'data/EXR/%s.%s.NOK.SP?format=sdmx-json&startPeriod=%s&endPeriod=%s&locale=en',
+    frequency, currency, start, end
+  )
+  req <- request(nbank_url) |> req_url_path_append(api_path)
```

Comment:
A minor style question: Should piping always be done with a new line? That is: 

```r
request(nbank_url) |> 
  req_url_path_append(api_path)
```

Reply:
I'm inclined to agree on this one, although I do like onliners 😅